packages/ds-component/src/types.ts

```diff
@@ -679,8 +679,8 @@ export interface ObjectRow {
 
 export interface ObjectTables {
   [TableType.DEFAULT]: ObjectRow[];
-  [TableType.COMPARISON]?: ObjectRow[][];
-  [TableType.SUMMARY]?: ObjectRow[][];
+  [TableType.COMPARISON]?: ObjectRow[];
```

Comment:
These types _might_ be right. I'd double check after updating based on my other comment.

Reply:
Pretty sure this is right. I don't remember why those would be different.